topo_processor/cli/geostore/delete.py

```diff
@@ -0,0 +1,45 @@
+import boto3
+import click
+from linz_logger import LogLevel, get_log, set_level
+
+from topo_processor.geostore.invoke import invoke_lambda
+from topo_processor.util.time import time_in_ms
+
+
+@click.command()
+@click.option(
+    "-id",
+    "--dataset-id",
+    required=True,
+    help="The dataset id to delete",
+)
+@click.option(
```

Comment:
its a good idea to have either a `--dry-run` or default to dry run and have a `--commit` before deleting things,

`delete.py  --dataset-id 12345 --commit` to me makes more sense than a  `delete.py --dataset-id 12345 -p`